src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,63 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import { MovieCard } from '../MovieCard';
+
 import './FindMovie.scss';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { ResponseError } from '../../types/ReponseError';
+
+type Props = {
+  onAddMovie: (newMovie: Movie) => void
+};
+
+export const FindMovie: React.FC<Props> = ({
+  onAddMovie,
+}) => {
+  const [query, setQuery] = useState<string>('');
+  const [
+    searchError,
+    setSearchError,
+  ] = useState<ResponseError | undefined>(undefined);
+  const [
+    searchedMovie,
+    setSearchedMovie,
+  ] = useState<Movie | undefined>(undefined);
+
+  const handleFindMovie = () => getMovie(query)
+    .then(data => {
+      if ('Error' in data) {
+        setSearchError(data);
+      } else {
+        if (searchError) {
+          setSearchError(undefined);
+        }
+
+        setSearchedMovie({
+          title: data.Title,
+          description: data.Plot,
+          imgUrl: data.Poster,
+          imdbUrl: `https://www.imdb.com/title/${data.imdbID}/`,
+          imdbId: data.imdbID,
+        });
+      }
+    });
+
+  const handleSearchMovie = (
+    e: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setQuery(e.target.value);
+  };
 
-export const FindMovie: React.FC = () => {
   return (
     <>
-      <form className="find-movie">
+      <form
+        onSubmit={(e) => {
+          e.preventDefault();
+          handleFindMovie();
+        }}
```

Comment:
```suggestion
        onSubmit={handleFindMovie}
```
it should accept `event`
do `preventDefault` there

you might rename a function to `handleSubmit` in this case